sag_desc.par

```diff
@@ -59,19 +59,19 @@ Charon's boathook
 /|
 /
 5-comma, syntonic comma (81/80)
-Didymus' dibbler
+Didymus' dibbler (Osiris' flail)
 '/|
 '/
-3-comma, Pythagorean comma (531441/524288)
+1-comma, Pythagorean comma (531441/524288)
 
 )/|
-�
-5:19 (or 13) -comma (41553/40960)
+
+5:19-comma (41553/40960)
 Hades' cornucopia
 |)
 f
 7-comma (64/63)
-Archytas' arc (or rainbow)
+Archytas' arc (Osiris' crook)

```

Comment:
It's Osiris's rainbow only when inverted, according to the character map.

Reply:
You mean Archytas' rainbow. It's the character map that is misleading. The intention is that it is both Archytas' arc or rainbow and Osiris' crook, no matter which way up it is. I believe it was done as it is in the character map to save space. 

You should change the entry for |) to "Archytas' arc (or rainbow) (Osiris' crook)". And if you think it best, you could remove the parentheses from "or rainbow" everywhere it occurs.

Reply:
Got it. Will do.